File: app/src/main/java/moe/kotohana/randomfood/utils/RandomHelper.java
package moe.kotohana.randomfood.utils;

import java.util.Random;

/**
 * Created by Junseok Oh on 2017-06-13.
 */

public class RandomHelper {
    public static int getRandomNumber(int range) {
        return new Random().nextInt(range + 1);
    }
    public static int getRandomNumber(int start, int range) {
        return new Random().nextInt(range + start);
    }
}

File: app/src/main/java/moe/kotohana/randomfood/utils/RandomHelper.kt
package moe.kotohana.randomfood.utils

import java.util.Random

/**
 * Created by Junseok Oh on 2017-06-13.
 */

class RandomHelper {
    companion object {
        fun getRandomNumber(range: Int): Int {
            return Random().nextInt(range + 1)
        }

        fun getRandomNumber(start: Int, range: Int): Int {
            return Random().nextInt(range + start)
        }
    }

}
